### PR TITLE
update blog cta

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr updates the right toc cta to encourage users to check out the coalesce 2023 sessions

### CODE REVIEW

File: website/blog/ctas.yml [hunk 1]
```diff
@@ -14,4 +14,9 @@
   header: Join data practitioners worldwide at Coalesce 2023
   subheader: Kicking off on October 16th, both online and in-person (Sydney, London, and San Diego)
   button_text: Register now
-  url: https://coalesce.getdbt.com/?utm_medium=internal&utm_source=docs&utm_campaign=q3-2024_coalesce-2023_aw&utm_content=coalesce____&utm_term=all___
\ No newline at end of file
+  url: https://coalesce.getdbt.com/?utm_medium=internal&utm_source=docs&utm_campaign=q3-2024_coalesce-2023_aw&utm_content=coalesce____&utm_term=all___
+- name: coalesce_2023_catchup
+  header: Missed Coalesce 2023?
+  subheader: Watch the full sessions and highlights from Coalesce 2023, the annual analytics engineering conference sponsored by dbt Labs.
```
Comment (mirnawong1) [inline website/blog/ctas.yml]:
```suggestion
  subheader: Watch Coalesce 2023 highlights and full sessions, dbt Labs' annual analytics engineering conference.
```

**Comment by mirnawong1** [comment]
shortened cta description as it looked too long

**Review by dbeatty10 — APPROVED** [review]
Looks good in lower right-hand corner [here](https://docs-getdbt-com-git-mwong-remove-coalesce-cta-dbt-labs.vercel.app/blog/announcing-materialized-views):

image

Also double-checked that the link to YouTube worked 👍

**Comment by mirnawong1** [comment]
thanks so much @dbeatty10 ! i also just shortened the descriptio n since it looked long:

> Watch Coalesce 2023 highlights and full sessions, dbt Labs' annual analytics engineering conference.

**Pull request merged**